go/client/chat_cli_rendering.go

```diff
@@ -24,14 +32,21 @@ func (v conversationInfoListView) show(g *libkb.GlobalContext) error {
 	ui := g.UI.GetTerminalUI()
 	w, _ := ui.TerminalSize()
 
+	prefixNumOfBytes, err := ConversationAliasPrefixNumOfBytes(v.aliases(), 2)
+	if err != nil {
+		return err
+	}
+
 	table := &flexibletable.Table{}
-	for i, conv := range v {
+	for _, conv := range v {
 		participants := strings.Split(conv.TlfName, ",")
 		table.Insert(flexibletable.Row{
 			flexibletable.Cell{
 				Frame:     [2]string{"[", "]"},
 				Alignment: flexibletable.Right,
-				Content:   flexibletable.SingleCell{Item: strconv.Itoa(i + 1)},
+				Content: flexibletable.SingleCell{
+					Item: string(MakeFullConversationAlias(conv.Id).MustShorten(prefixNumOfBytes)),
```

Comment:
haven't all of these already been calculated on line 35 by ConversationAliasPrefixNumOfBytes?  And MakeFullConversationAlias() already called in v.aliases().  Any way to reuse?